frontend/packages/dev-console/src/components/project-access/ProjectAccess.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {
+  Form,
+  TextInputTypes,
+  ActionGroup,
+  Alert,
+  Button,
+  ButtonVariant,
+} from '@patternfly/react-core';
+import { FormikProps, FormikValues } from 'formik';
+import { ButtonBar } from '@console/internal/components/utils';
+import { MultiColumnField, InputField, DropdownField } from '../formik-fields';
+
+enum accessRoles {
+  '' = 'Select a role',
+  admin = 'Admin',
+  view = 'View',
+}
+
+const ProjectAccess: React.FC<FormikProps<FormikValues>> = ({
+  handleSubmit,
+  handleReset,
+  isSubmitting,
+  status,
+  errors,
+  dirty,
+}) => {
+  return (
+    <Form onSubmit={handleSubmit}>
+      <div className="co-m-pane__form">
+        <MultiColumnField
+          name="projectAccess"
+          addLabel="Add Access"
+          headers={['Name', 'Role']}
+          emptyValues={{ user: '', role: '' }}
+          toolTip="Remove Access"
+        >
+          <InputField name="user" type={TextInputTypes.text} placeholder="Name" />
+          <DropdownField name="role" items={accessRoles} fullWidth />
+        </MultiColumnField>
+        <hr />
+        <ButtonBar
+          inProgress={isSubmitting}
+          errorMessage={status && status.submitError}
+          successMessage={status && status.success}
+        >
+          {!status && dirty && (
+            <Alert
+              isInline
+              className="co-alert"
+              variant="info"
+              title="You made changes to this page."
+            >
+              Click Save to save changes or Reload to cancel.
```

Comment:
This message is different from pipeline resources. This needs to be consistent with admin console and other forms. Please talk to UX. cc @serenamarie125 
I suggest creating a common shared component so that we don't duplicate this text anymore.
```
              title="The information on this page is no longer current."
            >
              Click Reload to update and lose edits, or Save Changes to overwrite.
```

Reply:
+1 on using a shared component, although it would be nice to include the improved wording in that common component!

Reply:
@debsmita1 please use the text on Brigid's mocks for the common component! 
```
You made changes to this page.
Click Save to save changes or Reload to cancel.

```

Reply:
@christianvogt Created a shared component https://github.com/openshift/console/pull/2886/commits/f2b45ceea2472b08533273842d445898ae7caee6